core/src/main/scala/cats/syntax/either.scala

```diff
@@ -27,4 +177,69 @@ final class EitherOps[A, B](val eab: Either[A, B]) extends AnyVal {
    * }}}
    */
   def toXor: A Xor B = Xor.fromEither(eab)
+
+  /**
+   * Transform the `Either` into a [[EitherT]] while lifting it into the specified Applicative.
+   *
+   * {{{
+   * scala> import cats.implicits._
+   * scala> val e: Either[String, Int] = Right(3)
+   * scala> e.toEitherT[Option]
+   * res0: cats.data.EitherT[Option, String, Int] = EitherT(Some(Right(3)))
+   * }}}
+   */
+  def toEitherT[F[_]: Applicative]: EitherT[F, A, B] = EitherT.fromEither(eab)
+}
+
+final class EitherObjectOps(val either: Either.type ) extends AnyVal {
```

Comment:
Not having the space triggers a ScalaStyle warning about having a space after tokens. I tried to turn it off by putting `scalastyle:off ensure.single.space.after.token` on the line above and on the same line but it still seemed to trigger. Not sure which other ScalaStyle rule it might be.